jormungandr-integration-tests/src/common/startup/configuration_builder.rs

```diff
@@ -1,7 +1,7 @@
 use crate::common::configuration::{
     genesis_model::{Fund, GenesisYaml, Initial, LinearFees},
     jormungandr_config::JormungandrConfig,
-    node_config_model::{Logger, NodeConfig, Peer},
+    node_config_model::{Log, NodeConfig, Peer},
```

Comment:
This could be a good time to get rid of the redundant carpal-inducing module name suffixes like `_model` and `_config` and `_config_model`. And why is `JormungandrConfig` separate from `NodeConfig`, anyway?

Reply:
Same as in https://github.com/input-output-hk/jormungandr/pull/627#discussion_r302217842

Reply:
we will need to provide a unified config down the line. Let fix the issue at hand first.